install/kubernetes/cilium/templates/hubble-cli.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.hubble.cli.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hubble-cli
+  labels:
+    k8s-app: hubble-cli
+  namespace: {{ .Release.Namespace }}
+spec:
+  containers:
+  - name: hubble-cli
+    image: quay.io/cilium/hubble:latest
+    command:
+      - sleep
+      - "3600"
+    imagePullPolicy: Always
+    volumeMounts:
+      - mountPath: {{ dir .Values.hubble.socketPath }}
+        name: hubble-sock-dir
+        readOnly: true
+{{- if .Values.hubble.tls.enabled }}
+      - mountPath: /var/lib/hubble-relay/tls
+        name: tls
+        readOnly: true
+{{- end }}
+  restartPolicy: Always
+{{- with .Values.hubble.relay.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | trim | nindent 8 }}
+{{- end }}
+{{- with .Values.hubble.relay.tolerations }}
+      tolerations:
+      {{- toYaml . | trim | nindent 8 }}
+{{- end }}
```

Comment:
I don't think this should share Relay toleratoin/nodeSelector, as it does not need to run on the same nodes as Relay.

Reply:
The idea here was to have the `hubble-cli` Pod "as close as the Relay Pod as possible". I agree that neither `nodeSelector` nor `tolerations` are essential though.